README.md

```diff
@@ -13,32 +13,60 @@ Describe your example and specify what it is demonstrating. Consider the
 following questions:
 
 * Is it pedagogical or a usable application?
-* Does it belong to a particular domain such as material simulation or logistics? 
-* What level of Ocean proficiency does it target (beginner, advanced, pro)? 
+* Does it belong to a particular domain such as material simulation or logistics?
+* What level of Ocean proficiency does it target (beginner, advanced, professional)?
```

Comment:
What's actually the distinction between "advanced" and "professional"?

Reply:
I have no idea & I don't think @thisac does either as we had a conversation about this a week or two ago.

Maybe someone else knows the secret? @randomir or @arcondello? 

If we don't already, it would be good to have a doc outlining the difference between the levels (if we plan on keeping them).

Reply:
We're currently investigating the demo filters in Leap and will most likely be updating this further after we've decided if, and how, we'd like to change the tags. This doesn't seem to have anything directly to do with those, though, so I'd suggest we change this to just include beginner and advanced (not sure either what the distinction would be between advanced and professional), or just leave it for now and rethink this in a future PR.